examples/CMakeLists.txt

```
@@ -49,3 +49,6 @@ foreach ( EXAMPLE ${EXAMPLES} )
     install( TARGETS ${TARGET_NAME} )
 
 endforeach()
+
+# install the julia controller example as well
+install( PROGRAMS simple_trixi_controller.jl DESTINATION share/libtrixi/examples/ )
```

```suggestion
install( FILES simple_trixi_controller.jl DESTINATION share/libtrixi/examples/ )
```
Otherwise, `+x` will be set on the file by [CMake](https://cmake.org/cmake/help/latest/command/install.html#programs):
> The `PROGRAMS` form is identical to the `FILES` form except that the default permissions for the installed file also include `OWNER_EXECUTE`, `GROUP_EXECUTE`, and `WORLD_EXECUTE`.

---

Thanks for being so attentive!